src/formatDate.js

```diff
@@ -51,6 +51,54 @@
 
 function formatDate(date, fromFormat, toFormat) {
   // write code here
+  const arrDate = date.split(`${fromFormat[3]}`);
+  const sliceFromFormate = fromFormat.slice(0, 3);
+  const sliceToFormate = toFormat.slice(0, 3);
```

Comment:
instead of adding this variables, you can iterate until `fromFormat.length - 1` and `toFormat.length - 1`